eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -221,17 +283,50 @@ function Add-IssueAssignees {
   }
 
   $uri = "$GithubAPIBaseURI/$RepoOwner/$RepoName/issues/$IssueNumber/assignees"
-  $assigneesAdditions = SplitMembers -membersString $Assignees
+  $assigneesAdditions = Split-GitHubAPIParameterMembers -membersString $Assignees
   $parameters = @{
     assignees = @($assigneesAdditions)
   }
 
   return Invoke-GitHubAPIPost -apiURI $uri -body $parameters -token $AuthToken
 }
 
+function Request-GithubPrReviewers {
```

Comment:
I suggest `Request-GitHubPullRequestReviewers`

Reply:
`Request` seemed consistent with you GitHub desribes the process of adding reviewers. And since its also an approved poweshell verb I though that just ideal. Perhaps it can be `Request-GitHubPullRequestReviewers`

Reply:
Isn't this function setting the set of reviewers? If that is the case then I'm not sure I understand why we would use the "Request" verb. Request to mean seems like we are retrieving the set of reviewers not updating or setting them.